index.js

```diff
@@ -7,12 +7,105 @@ function ask(questionText) {
   });
 }
 
+
 start();
 
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
+  console.log("\nLet's play a game where you make up a number and I try to guess it.")
+  let secretNumber = await ask("\nWhat is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
-  process.exit();
+  //secretNumber = the number user inputs for computer to guess.
+  function compGuess(max, min) {
+    //compGuess = Function for Computer generated guess
+    return (min) + Math.floor(Math.random() * (max - min))
+
+  }
```

Comment:
You can reuse this function to get a random computer number instead of defining a new function down on lines 62-64. Also, in order to get this function to return a number that has the min and max inclusive, you can you the expression from [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random#Getting_a_random_integer_between_two_values_inclusive). You just need to add 1 to your max-min expression.